tests/utils.py

```diff
@@ -27,8 +27,9 @@ def add_sh_fake_config(tmpdir):
     sh_config_path = os.path.join(tmpdir, 'scrapinghub.yml')
     with open(sh_config_path, 'w') as sh_config_file:
         sh_config_file.write('\n'.join([
-            "projects:", "  dev: 12345",
-            "images:", "  dev: registry/user/project",
+            "projects:", "  dev: 12345", "images:",
+            "  dev: registry/user/project",
+            "  xyz: registry/xyz/project",
```

Comment:
this looks kinda ugly, what about replacing it with 

```python
SH_CONFIG_FILE = """
projects:
  dev: ...
images:
  dev: ...
  xyz: ...
"""

def add_sh_fake_config(tmpdir):
    # ....
    with open(sh_config_path, 'w') as sh_config_file:
         sh_config_file.write(SH_CONFIG_FILE)
```

same for setup.py below